roles/uchiwa/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+ - firewall
+ - sensu/server
```

Comment:
I don't think we want this to depend on `sensu/server`, since we may have web uis running on a different host.  @paramite any opinion?  I've been doing my test installs with web services on a dedicated host...